### PR TITLE
Added question "But what universe is this?"

### PR DESCRIPTION
**Pull request by jhonnyjblaze**
@casey-collab how does this look to you?

### CODE REVIEW

File: english/universe-test/blaze_verse.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+But what universe is this?
+
+[This one?](https://www.youtube.com/watch?v=U5LVb3UtD3Q)
+
+Or...
+
+[Is it this one?](https://www.youtube.com/watch?v=tbSo7VmU7xo)
+
+Either way we are cool, [right?] (./marshmallow.md)
```
Comment (casey-collab) [inline english/universe-test/blaze_verse.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jhonnyjblaze** [comment]
@casey-collab good to go now?

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**